synthesis/synthesis.go

```diff
@@ -260,76 +181,61 @@ func FixCds(sqlitePath string, sequence string, codontable codon.Table, problema
 				if codon.Triplet != toCodon.Triplet {
 					toCodonBias := strings.Count(toCodon.Triplet, "G") + strings.Count(toCodon.Triplet, "C")
 					switch {
-					case codonBias == toCodonBias:
-						tx.MustExec(`INSERT INTO codonbias(fromcodon, tocodon, gcbias) VALUES (?, ?, ?)`, codon.Triplet, toCodon.Triplet, "NA")
 					case codonBias > toCodonBias:
-						tx.MustExec(`INSERT INTO codonbias(fromcodon, tocodon, gcbias) VALUES (?, ?, ?)`, codon.Triplet, toCodon.Triplet, "AT")
+						atBiasMap[codon.Triplet] = append(atBiasMap[codon.Triplet], toCodon.Triplet)
 					case codonBias < toCodonBias:
-						tx.MustExec(`INSERT INTO codonbias(fromcodon, tocodon, gcbias) VALUES (?, ?, ?)`, codon.Triplet, toCodon.Triplet, "GC")
+						gcBiasMap[codon.Triplet] = append(gcBiasMap[codon.Triplet], toCodon.Triplet)
 					}
+					naBiasMap[codon.Triplet] = append(naBiasMap[codon.Triplet], toCodon.Triplet)
 				}
 			}
 		}
+		// aminoAcidTotal can't equal zero or else division later will panic
```

Comment:
What does aminoAcidTotal do though? Where is it used and how?

Reply:
> Where is it used and how
`/aminoAcidTotal` for vim users or `ctrl+f aminoAcidTotal` for normies